library/ssl_tls13_keys.c

```diff
@@ -352,11 +179,25 @@ int mbedtls_ssl_tls13_hkdf_expand_label(
                                  hkdf_label,
                                  &hkdf_label_len );
 
-    return( psa_ssl_status_to_mbedtls(
-                mbedtls_psa_hkdf_expand( hash_alg,
-                                         secret, secret_len,
-                                         hkdf_label, hkdf_label_len,
-                                         buf, buf_len ) ) );
+    status = psa_key_derivation_setup( &operation, PSA_ALG_HKDF_EXPAND( hash_alg ) );
+    if (status == PSA_SUCCESS)
+        status |= psa_key_derivation_input_bytes( &operation,
+                                                  PSA_KEY_DERIVATION_INPUT_SECRET,
+                                                  secret,
+                                                  secret_len );
+    if (status == PSA_SUCCESS)
+        status |= psa_key_derivation_input_bytes( &operation,
+                                                  PSA_KEY_DERIVATION_INPUT_INFO,
+                                                  hkdf_label,
+                                                  hkdf_label_len );
+    if (status == PSA_SUCCESS)
+        status |= psa_key_derivation_output_bytes( &operation,
+                                                   buf,
+                                                   buf_len );
+    if (status == PSA_SUCCESS)
+        status |= psa_key_derivation_abort( &operation );
+
```

Comment:
Should we leave `hkdf_label` in memory, or should we zeroize it? I don't know how sensitive it is considered here.

Reply:
Fixed.